src/Microsoft.Azure.SignalR.Management/HubInstanceFactories/IInitializationTask.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.SignalR.Management
+{
+    internal interface IInitializationTask
```

Comment:
`IServiceConnectionInitializer`?

Reply:
For transient mode, there is no service connection, but a health checker.

Reply:
then `IInitializer`